pkg/util/utils.go

```diff
@@ -377,3 +377,16 @@ func ValidatePullType(pullType string) (PullType, error) {
 		return PullImageMissing, errors.Errorf("invalid pull type %q", pullType)
 	}
 }
+
+func ExitCode(err error) int {
+	if err == nil {
+		return 0
+	}
+	e := strings.ToLower(err.Error())
+	if strings.Contains(e, "file not found") ||
+		strings.Contains(e, "no such file or directory") {
```

Comment:
lol that's right, you tell that silly golang the proper way! :grin:

Reply:
Can we not use `os.IsNotExist`?

Reply:
I see this is what we were doing before and it horrifies me but I suppose there must be a reason...

Reply:
We are getting a string back from the OCI runtime that looks like.

ERRO[0024] container_linux.go:346: starting container process caused "exec: \"/etc1\": stat /etc1: no such file or directory": OCI runtime command not found error 

Which is why they have this horrendous code.

Reply:
We don't get ENOENT.